synapse/api/auth.py

```diff
@@ -209,6 +211,9 @@ def get_user_by_req(
             user_id, app_service = yield self._get_appservice_user_id(request)
             if user_id:
                 request.authenticated_entity = user_id
+                opentracing.set_tag("authenticated_entity", user_id)
+                # there is at least one other place where authenticated entity is
```

Comment:
I'm not sure this comment adds any value now

Reply:
True

Reply:
... so can you remove it please? :)